Main_Control_Node/src/LCD/LCD.c

```diff
@@ -0,0 +1,544 @@
+{
+  TransmitCommand(0x46);
+  TransmitCommandParameter(offset);
+  TransmitCommandParameter(0x48);
+  TransmitCommand(0x42);
+
+  for (int i = 0; i < BYTEPERBITMAP; i++)
+  {
+     TransmitCommandParameter(bitmap[i]);
+  }
+
+}
+
+
+/*
+ * Writes custom bitmaps to the LCD screen RAM
+ */
+void CharacterBitMaps(void)
+{
+  uint8_t bitmap[16][BYTEPERBITMAP] = {{0x7E, 0x7E, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x7E, 0x7E},
+                                    {0x7E, 0x7E, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x7E, 0x7E},
+                                    {0x7E, 0x7E, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60},
+                                    {0x7E, 0x7E, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06},
+                                    {0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x7E, 0x7E},
+                                    {0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x7E, 0x7E},
+                                    {0x7E, 0x7E, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+                                    {0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x7E, 0x7E},
+                                    {0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60, 0x60},
+                                    {0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06, 0x06},
+                                    {0x00, 0x06, 0x06, 0x0E, 0x0E, 0x1E, 0x36, 0x36, 0x36, 0x66, 0x66, 0x66, 0xE6, 0xE6, 0x86, 0x00},
+                                    {0x00, 0x62, 0x62, 0x62, 0x66, 0x66, 0x66, 0x6C, 0x6C, 0x6C, 0x70, 0x70, 0x70, 0x60, 0x60, 0x00},
+                                    {0x00, 0x60, 0x60, 0x70, 0x70, 0x78, 0x6C, 0x6C, 0x6C, 0x66, 0x66, 0x66, 0x67, 0x67, 0x61, 0x00},
+                                    {0x00, 0x46, 0x46, 0x46, 0x66, 0x66, 0x66, 0x36, 0x36, 0x36, 0x0E, 0x0E, 0x0E, 0x06, 0x06, 0x00},
+                                    {0xC0, 0xE0, 0x60, 0x30, 0x30, 0x30, 0x18, 0x18, 0x0C, 0x0C, 0x04, 0x06, 0x06, 0x06, 0x07, 0x03},
+                                    {0x03, 0x07, 0x06, 0x06, 0x0C, 0x0C, 0x18, 0x18, 0x18, 0x30, 0x30, 0x60, 0x60, 0x60, 0xE0, 0xC0}};
+  TransmitCommand(0x5C);
+  TransmitCommandParameter(0x00);
+  TransmitCommandParameter(0x40);
+  TransmitCommand(0x4C);
+
+  for (uint8_t i = 0; i < 16; i++)
+  {
+    SaveCharacterToRAM(bitmap[i], i*BYTEPERBITMAP);                
+  }   
+}
+
+/* Outputs a string of characters to the screen
+ * @Param Str[]: A string, "array of characters", whatever
+ * @Param starting_x: x-coordinate to write the string to
+ * @Param starting_y: y-coordinate to write the string to
+ * Returns: nothing

```

Comment:
can probably remove this since it is implied that nothing is returned if documentation is consistent